bot/bot.py

```diff
@@ -223,81 +58,26 @@ def insert_item_into_filter_list_cache(self, item: Dict[str, str]) -> None:
             "updated_at": item["updated_at"],
         }
 
-    async def login(self, *args, **kwargs) -> None:
-        """Re-create the connector and set up sessions before logging into Discord."""
-        # Use asyncio for DNS resolution instead of threads so threads aren't spammed.
-        self._resolver = aiohttp.AsyncResolver()
-
-        # Use AF_INET as its socket family to prevent HTTPS related problems both locally
-        # and in production.
-        self._connector = aiohttp.TCPConnector(
-            resolver=self._resolver,
-            family=socket.AF_INET,
-        )
+    async def cache_filter_list_data(self) -> None:
+        """Cache all the data in the FilterList on the site."""
+        full_cache = await self.api_client.get('bot/filter-lists')
 
-        # Client.login() will call HTTPClient.static_login() which will create a session using
-        # this connector attribute.
-        self.http.connector = self._connector
+        for item in full_cache:
+            self.insert_item_into_filter_list_cache(item)
 
-        self.http_session = aiohttp.ClientSession(connector=self._connector)
-        self.api_client = api.APIClient(connector=self._connector)
+    async def setup_hook(self) -> None:
+        """Default Async initialisation method for Discord.py."""
+        await super().setup_hook()
 
         if self.redis_session.closed:
             # If the RedisSession was somehow closed, we try to reconnect it
             # here. Normally, this shouldn't happen.
             await self.redis_session.connect()
```

Comment:
Why isn't this in bot-core instead?

Reply:
Good point, moved in 21acc03ca5a053881a529ed21e437060205684ac and https://github.com/python-discord/bot-core/pull/61